var/spack/repos/builtin/packages/kassiopeia/package.py

```diff
@@ -62,7 +63,10 @@ def cmake_args(self):
         if "+root" in self.spec:
             cxxstd = self.spec["root"].variants["cxxstd"].value
         else:
-            cxxstd = "14"
+            if "@:3.8.1" in self.spec:
```

Comment:
We generally recommend using `self.spec.satisfies()` for range checks.

```suggestion
            if self.spec.satisfies("@:3.8.1"):
```